pkg/httpserver/crud/create.go

```diff
@@ -2,25 +2,28 @@ package crud
 
 import (
 	"context"
-	"errors"
-	"net/http"
+	"time"
 
 	"github.com/gin-gonic/gin"
-	"github.com/justtrackio/gosoline/pkg/db"
+	"github.com/justtrackio/gosoline/pkg/cfg"
+	"github.com/justtrackio/gosoline/pkg/exec"
 	"github.com/justtrackio/gosoline/pkg/httpserver"
 	"github.com/justtrackio/gosoline/pkg/log"
-	"github.com/justtrackio/gosoline/pkg/validation"
 )
 
 type createHandler struct {
-	logger      log.Logger
-	transformer CreateHandler
+	logger       log.Logger
+	transformer  CreateHandler
+	releaseDelay time.Duration
 }
 
-func NewCreateHandler(logger log.Logger, transformer CreateHandler) gin.HandlerFunc {
+func NewCreateHandler(config cfg.Config, logger log.Logger, transformer CreateHandler) gin.HandlerFunc {
+	releaseDelay := config.GetDuration("crud.release_delay", 10*time.Minute)
```

Comment:
Please create a `settings.go` where this is a field on a struct, allowing us to
- have a single place for the default
- have a place to document what this actually does
- you could even implement it such that there is a delay per CRUD method with some general default if you wanted to (but maybe that is too much?)

Reply:
Added settings and documentation